src/WebJobs.Script/Config/FunctionsHostingConfigOptions.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    public class FunctionsHostingConfigOptions
+    {
+        private readonly Dictionary<string, string> _features;
+
+        public FunctionsHostingConfigOptions()
+        {
+            _features = new Dictionary<string, string>();
+        }
+
+        /// <summary>
+        /// Gets all features in the hosting configuration.
+        /// </summary>
+        public Dictionary<string, string> Features => _features;
+
+        /// <summary>
+        /// Gets a value indicating whether worker concurency feature is enabled in the hosting config.
+        /// </summary>
+        public bool FunctionsWorkerDynamicConcurrencyEnabled
```

Comment:
can you add feature flag for target based scaling as well?

Reply:
For TBS it's not make sense to add the properties as the flags are dynamic (assembly names). First step is getting assembly name where a target scalers is defined and then checking the config if the flag exists.